### PR TITLE
update registry

### PR DESCRIPTION
**Pull request by eyurtsev**
Potential registry update

### CODE REVIEW

File: langchain_benchmarks/rag/evaluators.py [hunk 1]
```diff
@@ -77,18 +77,21 @@ def evaluate_run(
...
-RAG_EVALUATION = RunEvalConfig(
-    evaluators=[
-        RunEvalConfig.LabeledScoreString(
-            criteria=_ACCURACY_CRITERION, llm=eval_llm, normalize_by=10.0
-        ),
-        RunEvalConfig.EmbeddingDistance(),
-    ],
-    custom_evaluators=[FaithfulnessEvaluator(llm=faithfulness_eval_llm)],
-)
+def get_evaluator() -> RunEvalConfig:
```
Comment (hinthornw) [inline langchain_benchmarks/rag/evaluators.py:81]: Ya nice i like this better

**Comment by eyurtsev** [comment]
Do not merge this one

File: langchain_benchmarks/registration.py [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+            for env in self.tasks
+        ]
+        return tabulate(table, headers=headers, tablefmt="html")
+
+    def __getitem__(self, key: Union[int, str]) -> Task:
+        """Get an environment from the registry."""
+        if isinstance(key, slice):
+            raise NotImplementedError("Slicing is not supported.")
+        elif isinstance(key, (int, str)):
+            # If key is an integer, return the corresponding environment
```
Comment (hinthornw) [inline langchain_benchmarks/registration.py:67]:
```suggestion
            # Fetch the name of the environment by name or index
```

**Comment by eyurtsev** [comment]
Closing in favor of other schema changes

**Pull request closed without merging**